src/SignalR/samples/JwtSample/Startup.cs

```diff
@@ -17,9 +18,16 @@ namespace JwtSample
 {
     public class Startup
     {
-        private readonly SymmetricSecurityKey SecurityKey = new SymmetricSecurityKey(Guid.NewGuid().ToByteArray());
+        private readonly SymmetricSecurityKey SecurityKey;
         private readonly JwtSecurityTokenHandler JwtTokenHandler = new JwtSecurityTokenHandler();
 
+        public Startup()
+        {
+            var key = new byte[16];
```

Comment:
Stackalloc 😄

Reply:
Oh wait, it's being stored in the key 😄

Reply:
Looking at this code makes me wonder if we need a dirt-simple API like:

```cs
byte[] bytes = RandomNumberGenerator.GetBytes(16);
```

Reply:
I was looking for an API like that 😄

Reply:
API proposal up at https://github.com/dotnet/runtime/issues/42769